sdk/batch/azure-mgmt-batch/sdk_packaging.toml

```diff
@@ -5,3 +5,4 @@ package_doc_id = "batch"
 is_stable = true
 sample_link = ""
 title = "BatchManagementClient"
+auto_update = false
```

Comment:
Please don't add this line in MGMT SDK config otherwise it may influence SDK release.

Reply:
The problem is if I don't disable it, the script updates READMEs automatically.

You can check https://github.com/Azure/azure-sdk-for-python/pull/30916/commits/61411540ab19873aa765a26134143dbb59d81475#diff-276cb3d38db08bd1067333f251899a01d57c7c5b695d2d2cd655e9af2a56ef24 as a reference.

It seems to me your mgmt SDKs bypass the script too?